docusaurus/website/siteConfig.js

```diff
@@ -68,7 +68,7 @@ const siteConfig = {
   },*/
 
     // This copyright info is used in /core/Footer.js and blog rss/atom feeds.
-    copyright: "Copyright © " + new Date().getFullYear() + " Tokopedia OSS",
+    copyright: "Copyright © " + new Date().getUTCFullYear() + " Tokopedia OSS",
```

Comment:
I'm afraid this is not solving the issues itself. Actually `getUTCFullYear()` and `getFullYear()` only differs in matter of timezone only. So it should be showing 2019 by now.

My guess is the docs page will only fetch new time when the config is instantiated. However, the config is only instantiated when the build process is done (but not for React.Component).

Therefore, I think it is better to create template on it like `{currentDate}` and replace it in `Footer` component. Because Footer will always be instantiated when user open the page.

Reply:
I see, but is it good idea to get globally date data (not from server) like from http://worldclockapi.com/? It's just to assure the data is keep up date.

Reply:
What do you mean globally?

I think it is better if we convert the date into timezone we want (recommend Western Indonesia Time) in the component itself rather than get it from API. So, the definition of "up-to-date" is based on our chosen timezone. But I think that would be too much since we only obtain the year :)